qa/L0_sequence_batcher/test.sh

```diff
@@ -82,7 +82,7 @@ TF_VERSION=${TF_VERSION:=1}
 # On windows the paths invoked by the script (running in WSL) must use
 # /mnt/c when needed but the paths on the tritonserver command-line
 # must be C:/ style.
-if [[ "$(< /proc/sys/kernel/osrelease)" == *Microsoft ]]; then
```

Comment:
This as well needs to be changed for gitlab side

Reply:
See the updated MR